utils/elastic_search/elastic_search_utils.py

```diff
@@ -23,27 +23,51 @@ def build_search_query(self, search_text: str, sort_by: str, order: str, search_
         """
 
         # Default sort order, if nothing is passed
-        sort_order = ['id', 'name', 'symbol']
+        sort_order = ['transaction_timestamp', 'id', 'name', 'symbol']
 
         if not sort_by or sort_by == 'uid':
             sort_by = 'id'
 
         sort_fields_data_type = {
-            'id': 'keyword',
-            'name': 'keyword',
-            'symbol': 'keyword'
+            'transaction_timestamp': {
+                'type': 'long',
+                'append_data_type_on_sort': False
+            },
+            'id': {
+                'type': 'keyword',
+                'append_data_type_on_sort': True
+            },
+            'name': {
+                'type': 'keyword',
+                'append_data_type_on_sort': True
+            },
+            'symbol': {
+                'type': 'keyword',
+                'append_data_type_on_sort': True
+            }
         }
 
         if not order:
             order = 'asc'
 
         primary_sort_key = {}
-        primary_sort_key[sort_by+'.'+sort_fields_data_type[sort_by]] = order
+
+        sort_by_complement = ''
+        if sort_fields_data_type[sort_by]['append_data_type_on_sort']:
+            sort_by_complement = '.'+str(sort_fields_data_type[sort_by]['type'])
+
+        primary_sort_key[sort_by+sort_by_complement] = order
```

Comment:
Do we have to append the data type on any type other than `keyword`?
If not this would feel a bit awkward and we could just append when the type is `keyword`.

Reply:
I am not sure if other data types will need to be appended or not, this is why I wanted to make as flexible as possible. Do you think we can keep this way?